app/categories/Categories.jsx

```diff
@@ -2,8 +2,6 @@ import React, { PureComponent, PropTypes } from 'react';
 import { post } from 'axios';
 import { Link } from 'react-router';
 
-import Spinner from '../components/Spinner';
```

Comment:
Spinner isn't being used anywhere right now so we can probably pitch it. The site is too fast for it.